packages/compass-preferences-model/src/preferences.ts

```diff
@@ -227,6 +189,7 @@ function featureFlagToPreferenceDefinition(
       featureFlag.stage === 'released'
         ? () => ({ value: true, state: 'hardcoded' })
         : undefined,
+    validator: Joi.boolean().default(false),
```

Comment:
For my own understanding: `.default()` implies `.optional()`, right?

Reply:
Yes. it does.